tasks/main.yml

```diff
@@ -46,21 +62,40 @@
     state: link
   when: "ansible_service_mgr != 'systemd'"
 
+- name: Set RUN_AS user to sonar, to avoid ElasticSearch failing with 'cannot run as root'
+  lineinfile:
+    path: /usr/local/sonar/bin/linux-x86-64/sonar.sh
+    firstmatch: yes
+    regexp: '^#RUN_AS_USER='
+    line: 'RUN_AS_USER=sonar'
+
 - name: Copy SonarQube systemd unit file into place (for systemd systems).
   template:
     src: sonar.unit.j2
     dest: /etc/systemd/system/sonar.service
     owner: root
     group: root
-    mode: 0755
+    mode: 0644
   when: "ansible_service_mgr == 'systemd'"
 
 - name: Ensure Sonar is running and set to start on boot.
-  service: name=sonar state=started enabled=yes
+  service: 
+    name: sonar
+    state: started
+    enabled: yes
+
+- name: "Ensure SonarQube files permissions"
+  file:
+    path: /usr/local/sonar
+    state: directory
+    recurse: yes
+    owner: "{{ sonar_user }}"
+    group: "{{ sonar_group }}"
+    mode: "go-w"
```

Comment:
This is more a point for discussion in the team but I would definitely prefer "number" mode statements.. `go-w` doesn't make much sense to me 🔸

Reply:
yeah, this is true, it was a copy pasta from the second role from me. I will amend.